transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestBodyHandler.java

```diff
@@ -90,6 +91,11 @@ private RestBodyHandler(boolean handleFileUploads, String uploadDirectory) {
     setUploadsDirectory(uploadDirectory);
   }
 
+  @VisibleForTesting
```

Comment:
we can set `VisibleForTesting` default to package visible, how do you think so?

Reply:
@liubao68 what do you mean, remove the `public` modifier?

Reply:
yes